src/openseadragon.js

```diff
@@ -209,6 +209,13 @@
   *     You can pass a CSS color value like "#FF8800".
   *     When passing a function the tiledImage and canvas context are available as argument which is useful when you draw a gradient or pattern.
   *
+  * @property {Object} [subPixelRounding=null]
+  *     Determines when subpixel rounding should be applied for tiles rendering.
+  *     This property is a subpixel rounding enum values dictionary [number] --> string.
+  *     The key is a $.BROWSERS value, and the value is one of 'ALWAYS', 'ONLY_AT_REST' or 'NEVER',
```

Comment:
The `$` is just a convenience inside the code; it's not something mentioned in the documentation. Instead it should be `OpenSeadragon`, though we generally don't mention the top-level namespace when referring to things. What would be good to do is to link to BROWSERS (http://openseadragon.github.io/docs/OpenSeadragon.html#.BROWSERS). That would look like `The key is a {@link BROWSERS} value`.